WorkflowDevelopment/T270/CodeSnippets/Activity1.5/Step1/RSSVWorkOrderWorkflow.cs

```diff
@@ -74,7 +74,7 @@ public override void Configure(PXScreenConfiguration config)
             var context = config.GetScreenConfigurationContext<RSSVWorkOrderEntry,
                 RSSVWorkOrder>();
 			
-            ///////// The added line
+            // Create an instance of the Conditions class
```

Comment:
@MehranQadri , if you display this in a guide, you need to surround the phragment you display with "the modified code-the end of modified code" comments.

Reply:
Fixed in the latest commit alongside other snippets.